www/components/messagebox.tsx

```diff
@@ -98,6 +164,9 @@ export default function MessageBox({
             </button>
           </div>
         )}
+        {isReactionPending && (
+          <p className="text-sm text-gray-500">Saving reaction...</p>
```

Comment:
Can you use a loading spinner on the image instead of adding additional text. Would be more space efficient. 

Asked claude for a quick component for that and got this

```jsx
import React from 'react';

const Spinner = ({ size = 40, color = '#000000' }) => {
  const spinnerStyle = {
    width: `${size}px`,
    height: `${size}px`,
    border: `4px solid ${color}`,
    borderTop: `4px solid transparent`,
    borderRadius: '50%',
    animation: 'spin 1s linear infinite',
  };

  return (
    <div style={{ display: 'inline-block' }}>
      <style>
        {`
          @keyframes spin {
            0% { transform: rotate(0deg); }
            100% { transform: rotate(360deg); }
          }
        `}
      </style>
      <div style={spinnerStyle}></div>
    </div>
  );
};

export default Spinner;
```

Reply:
Might be able to just use a `Fa` icon as well and add the keyframe spin.